src/frontend/src/layouts/PageBlock.tsx

```diff
@@ -28,12 +29,19 @@ interface PageBlockProps {
  * @param style Optional styling for the pageblock
  * @param defaultClosed Sets the pageblock to be closed (collapsed) by default.
  */
-const PageBlock: React.FC<PageBlockProps> = ({ title, headerRight, children, style, defaultClosed }) => {
+const PageBlock: React.FC<PageBlockProps> = ({
+  title,
+  headerRight,
+  children,
+  style,
+  defaultClosed,
+  transparentBackground
```

Comment:
I see what you're doing here. Not saying I don't like it, but remove it from this PR. If you really wanna do smthn like this, talk to Zach, come up with a design that looks good, and make an epic with a comprehensive plan for replacing and/or updating everywhere with a pageblock. I don't wanna randomly change it in one place like this cause that's how we end up with the site looking poorly organized/ random. I really wanna move towards the site having a clean, cohesive look this upcoming semester